pkg/release/release.go

```diff
@@ -116,6 +119,12 @@ func readReleases(branch string) ([]ReleaseObject, error) {
 	var releases []ReleaseObject
 	{
 		for _, v := range r.Resources {
+			// skip legacy (< 10)
+			versionParts := strings.Split(strings.TrimPrefix(v, "v"), ".")
+			if i, _ := strconv.Atoi(versionParts[0]); i < firstAWSNodePoolsReleaseMajor {
```

Comment:
Do we want to skip error handling here?

Reply:
Ideally not, but the yaml file from the releases repository is protected to have sane version numbers by its CI.

Error handling could look like this:

```suggestion
			if i, err := strconv.Atoi(versionParts[0]); i < firstAWSNodePoolsReleaseMajor || err != nil {
```

Reply:
It's up to you, but I'd rather have the extra safety net. We shouldn't rely on another repo to provide a valid input

Reply:
Ok, then lets go with error handling :+1: